noetic/distribution.yaml

```diff
@@ -6928,6 +6928,12 @@ repositories:
       url: https://github.com/ros-visualization/visualization_tutorials.git
       version: noetic-devel
     status: maintained
+  vrpn:
+    release:
```

Comment:
Please also add a `source` entry (and consider a `doc` entry).

Reply:
Done!